hpedockerplugin/hpe/volume.py

```diff
@@ -24,7 +24,7 @@ def createvol(name, size=DEFAULT_SIZE, prov=DEFAULT_PROV,
               rcg_info=None):
     volume = {}
     volume['id'] = str(uuid.uuid4())
-    volume['name'] = volume['id']
+    volume['name'] = name
```

Comment:
@sonivi @wdurairaj this removes the duplicate entry for volume ID in a comment field but creates a duplicate entry for docker volume name. Also in our code, we need to make sure that nowhere we have used volume['name'] to get the uuid of a volume, else it will break the functionality.

#224 referes to - 
Volume id appears twice as below
dcv-cWAi04kgRPmt5hU6S3BmyQ {"volume_id": "716022d3-8920-44f9-ade6-153a4b7066c9", "name": "716022d3-8920-44f9-ade6-153a4b7066c9", "type": "Docker", "display_name": "DOCKER-VOL"}

but doing below changes will make the output look like below:
dcv-cWAi04kgRPmt5hU6S3BmyQ {"volume_id": "716022d3-8920-44f9-ade6-153a4b7066c9", "name": "DOCKER-VOL", "type": "Docker", "display_name": "DOCKER-VOL"}

Reply:
@nilangekarss , i was thinking of doing something like:
volume['name'] = utils.get_3par_vol_name(volume['id'])

I agree with you need to check its impact to avoid breaking of any existing functionality